tensorflow/python/layers/normalization_test.py

```diff
@@ -68,19 +68,22 @@ def _train(self,
              use_gpu,
              is_fused,
              restore=False,
-             freeze_mode=False):
+             freeze_mode=False,
+             dtype=None):
```

Comment:
Have dtype default to dtypes.float32. Then you don't have to have the "if dtype is None" check below.

Reply:
Done.